common/package.json

```diff
@@ -28,6 +28,7 @@
     "@types/uuid": "^8.3.0",
     "@weco/toggles": "1.0.0",
     "@wojtekmaj/enzyme-adapter-react-17": "^0.6.3",
+    "axios": "^0.21.4",
```

Comment:
Was this added for something in particular? I prefer `axios`, but we've tended to use (the horrifically titled) `isomorphic-unfetch` for most things

Reply:
It was being used by a component in common (I think user info related) but wasn't installed as a dep (I think I forgot to add the dep when I extracted said component from the identity app) so caused the build to fail when it was separated out more strictly. Indeed we should use one for everything, I'll do that separately as well.

Reply:
if the one thing for everything could be `axios` that'd be nice… (probably too much of a rewrite of too many things at this point, though).

Reply:
Interesting, looks like Next [now provides a polyfill for fetch](https://nextjs.org/docs/basic-features/supported-browsers-features#server-side-polyfills)